rest/user_organizations.php

```diff
@@ -48,7 +48,7 @@
         }
 
     } else {
-	    $user_organization->user_id = $data['user_id'];
+	    // $user_organization->user_id = $data['user_id'];
```

Comment:
Why is setting the user_id commented out?

Reply:
`could not read key 'user_id' from $data`

Reply:
We're going to need the user_id though, or we can't create the row. It sounds like the problem is the code calling the service not setting it.

Reply:
I have un-commented this line and it didn't cause any errors when creating or editing users so I don't think it is causing an error any more.